src/shared_utils/shared_utils.star

```diff
@@ -71,3 +73,51 @@ def label_maker(client, client_type, image, connected_client, extra_labels):
     }
     labels.update(extra_labels)  # Add extra_labels to the labels dictionary
     return labels
+
+
+def get_devnet_enodes(plan, filename):
+    enode_list = plan.run_python(
+        files={constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: filename},
+        wait=None,
+        run="""
+with open("/network-configs/network-configs/bootnode.txt") as bootnode_file:
+    bootnodes = []
+    for line in bootnode_file:
+        line = line.strip()
+        bootnodes.append(line)
+print(",".join(bootnodes), end="")
+            """,
+    )
+    return enode_list.output
+
+
+def get_devnet_enrs_list(plan, filename):
+    enr_list = plan.run_python(
+        files={constants.GENESIS_DATA_MOUNTPOINT_ON_CLIENTS: filename},
+        wait=None,
+        run="""
+with open("/network-configs/network-configs/bootstrap_nodes.txt") as bootnode_file:
+    bootnodes = []
+    for line in bootnode_file:
+        line = line.strip()
+        bootnodes.append(line)
+print(",".join(bootnodes), end="")
+            """,
+    )
+    return enr_list.output
+
+
+# Prysm and Nimbus needs to have the enrs in a list format
```

Comment:
but are they happy with 

"apple, mango, cat"

that should be do-able. I doublt they require a "list" type; a string should be enough?

Reply:
Lets do this after both this and #437 is merged in.